### DISCUSSION TITLE
Relationship between giu and cimgui-go

### DISCUSSION THREAD

**Discussion by viduq** [AllenDang/giu#749] [General]
Hi and thank you for creating giu!
I am wondering what the relationship between giu and cimgui-go is. Will cimgui-go replace giu? Will giu use cimgui-go under the hood?
When starting a new project, which one is to prefer? Is it possible to use code written for giu also with cimgui-go?

**Comment by AllenDang** [comment]
Giu is a high-level wrapper of cimgui-go and it is using cimgui-go under the hood.
For me, I will use giu to create GUI, because it is much convenient.

**Reply by viduq** [reply]
Thanks for the reply.
